requirements.txt

```diff
@@ -32,3 +32,5 @@ requests~=2.32.3
 httpx~=0.27.2
 typing_extensions~=4.12.2
 asgiref~=3.8.1
+
+djangorestframework~=3.15.2
```

Comment:
If we can use the `api_client` fixture from pytest-django, we shouldn't need this I believe.
```suggestion
```

Reply:
@SandyRogers please see my comment here:

https://github.com/EBI-Metagenomics/emgapi-v2/pull/27#discussion_r1907131347